### PR TITLE
Document changes for Bot API 7.0

### PR DESCRIPTION
**Pull request by KnorpelSenf**
https://core.telegram.org/bots/api

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Ready for review now, sorry for the many fixes

File: site/docs/guide/context.md [hunk 1]
```diff
@@ -169,12 +179,12 @@ This can be used to pass further configuration to every API call.
 ::: tip Telegram Reply Feature
 Even though the method is called `ctx.reply` in grammY (and many other frameworks), it does not use the [reply feature of Telegram](https://telegram.org/blog/replies-mentions-hashtags#replies) where a previous message is linked.
 
-If you look up what `sendMessage` can do in the [Telegram Bot API Reference](https://core.telegram.org/bots/api#sendmessage), you will see a number of options, such as `parse_mode`, `disable_web_page_preview`, and `reply_to_message_id`.
+If you look up what `sendMessage` can do in the [Telegram Bot API Reference](https://core.telegram.org/bots/api#sendmessage), you will see a number of options, such as `parse_mode`, `link_preview_options`, and `reply_parameters`.
```
Comment (roj1512) [inline site/docs/guide/context.md:182]:
```suggestion
If you look up what `sendMessage` can do in the [Bot API Reference](https://core.telegram.org/bots/api#sendmessage), you will see a number of options, such as `parse_mode`, `link_preview_options`, and `reply_parameters`.
```

File: site/docs/guide/basics.md [hunk 2]
```diff
@@ -55,22 +55,25 @@ Also, check out the [next section](./context) to learn how the context object of
...
 });
 ```
 
 > Note that only sending a message via `ctx.reply` does **NOT** mean you are automatically replying to anything.
-> Instead, you should specify `reply_to_message_id` for this.
+> Instead, you should specify `reply_parameters` for this.
 > The function `ctx.reply` is just an alias for `ctx.api.sendMessage`, see the [next section](./context#available-actions).
 
+The reply parameters also allow you to reply to messages in other chats, as well as to quote parts of a message---or even both at the same time!
+Have a look at the [documentation](https://core.telegram.org/bots/api#replyparameters) that Telegram provides for reply parameters.
```
Comment (roj1512) [inline site/docs/guide/basics.md:75]:
```suggestion
Have a look at Bot API's [reply parameters documentation](https://core.telegram.org/bots/api#replyparameters).
```

File: site/docs/guide/context.md [hunk 3]
```diff
@@ -89,8 +90,17 @@ bot.on("message:entities", async (ctx) => {
...
+
+bot.on("message_reaction", (ctx) => {
+  const { emojiAdded } = ctx.reactions();
+  if (emojiAdded.includes("🎉")) {
+    await ctx.reply("partY");
+  }
+});
 ```
 
+> Skip ahead to [reactions](./reactions.md) if you are interested in them.
```
Comment (roj1512) [inline site/docs/guide/context.md:102]:
```suggestion
> Skip ahead to [Reactions](./reactions.md) if you are interested in them.
```

File: site/docs/guide/reactions.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
+# Reactions
+
+Bots can work with message reactions.
+There are two types of reactions, emoji reactions and custom emoji reactions.
```
Comment (roj1512) [inline site/docs/guide/reactions.md:4]:
```suggestion
There are two types of reactions: emoji reactions and custom emoji reactions.
```

File: site/docs/guide/reactions.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+## Reacting to Messages
+
+Bots can add a single emoji reaction to a message.
+If a [Telegram Premium](https://telegram.org/faq_premium?setln=en) user added a custom emoji reaction to a message, bots can add the same reaction to this message, too.
```
Comment (roj1512) [inline site/docs/guide/reactions.md:9]: > A custom emoji reaction can be used if it is either already present on the message or **_explicitly allowed by chat administrators_**.\[1\]

[1]: https://core.telegram.org/bots/api#setmessagereaction ^reaction parameter description

File: site/docs/guide/reactions.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+// Use `ctx.api.setMessageReaction` for reactions somewhere else.
+bot.on("message", async (ctx) => {
+  await ctx.api.setMessageReaction(chat_id, message_id, "🎉");
+});
+
+// Use `bot.api.setMessageReaction` outside handlers.
+await bot.api.setMessageReaction(chat_id, message_id, "💯");
+```
+
+As usualy, TypeScript will auto-complete the possible emoji for you.
```
Comment (roj1512) [inline site/docs/guide/reactions.md:27]:
```suggestion
As usual, TypeScript will provide an auto-complete for the emojis you can use.
```

File: site/docs/guide/reactions.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+The list of available emoji reactions can be found [here](https://core.telegram.org/bots/api#reactiontypeemoji).
+
+::: tip Emoji Plugin
+It can be ugly to program with emoji.
+Not all systems can display your source code properly.
+Also, it is annoying to copy them from Google all the time.
```
Comment (roj1512) [inline site/docs/guide/reactions.md:33]:
```suggestion
Also, it is annoying to copy them from different places all the time.
```

File: site/docs/plugins/emoji.md [hunk 8]
```diff
@@ -99,6 +99,19 @@ bot.command("ping", async (ctx) => {
 If you're unfamiliar with this syntax, you can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).
 :::
 
+## Useful Data for Reactions
+
+When you use [reactions](../guide/reactions.md) with your bot, you will have to program a lot with emoji, too!
+This is equally annoying, and since this plugin is the wet dream of all your emoji usage, it can help you with reactions, too.
```
Comment (roj1512) [inline site/docs/plugins/emoji.md:105]: 🤩

File: site/docs/guide/reactions.md (hunk 5 above)
Comment (roj1512) [reply site/docs/guide/reactions.md:9]:
```suggestion
Bots can add a single emoji reaction to a message.
They can add custom emoji reactions only if it is explicitly allowed by the chat's administrators, or there is a custom emoji reaction previously added by a subscriber of [Telegram Premium](https://telegram.org/faq_premium?setln=en).
```

File: site/docs/guide/reactions.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+bot.on("message", async (ctx) => {
+  await ctx.api.setMessageReaction(chat_id, message_id, "🎉");
+});
+
+// Use `bot.api.setMessageReaction` outside handlers.
+await bot.api.setMessageReaction(chat_id, message_id, "💯");
+```
+
+As usualy, TypeScript will auto-complete the possible emoji for you.
+The list of available emoji reactions can be found [here](https://core.telegram.org/bots/api#reactiontypeemoji).
```
[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/guide/reactions.md:28]: It's _the_ list, the definitive source of truth for available reactions—not just any list.

File: site/docs/guide/reactions.md (hunk 7 above)
Comment (KnorpelSenf) [reply site/docs/guide/reactions.md:33]: nice, thanks

File: site/docs/guide/reactions.md (hunk 5 above)
Comment (KnorpelSenf) [reply site/docs/guide/reactions.md:9]: Right, I'll clarify it.

**Review by roj1512 — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
Sorry, I do not wish to wait.

**Pull request merged**